psycop/common/feature_generation/loaders/raw/load_text.py

```diff
@@ -149,6 +149,7 @@ def load_text_sfis(
 def load_text_split(
     text_sfi_names: str | Iterable[str],
     split_name: Sequence[SplitName],
+    split_type: Literal["original", "geographical"] = "original",
```

Comment:
Ah, this is probably why you want to wrap the two functions. Instead, perhaps the split here should be a function? (Dependency-injected).